deployment/helm/values.yaml

```diff
@@ -17,8 +20,11 @@ deployAlertWithBlackDuck: false
 enableStandalone: true
 cfssl:
   image: "docker.io/blackducksoftware/blackduck-cfssl:1.0.0"
-  limitMemory: "640M"
-  requestMemory: "640M"
+  resources:
+    limits:
+      memory: "640M"
+    requests:
+      memory: "640M"
```

Comment:
Same bug as above 😄

Reply:
Also, I'm not sure why we use 640, whereas the docker-compose file specifies 512: https://github.com/blackducksoftware/blackduck-alert/blob/5.2.1/deployment/docker-compose/standalone/docker-compose.yml#L13

Reply:
Interesting, so this may also be a bug, since Black Duck also specifies 512 for the cfssl container: https://github.com/blackducksoftware/hub/blob/v2019.12.1/docker-compose/docker-compose.yml#L107

Reply:
Nice find! I switched it to Mi

Reply:
Interesting that it's 640 in the docker-swarm file: https://github.com/blackducksoftware/blackduck-alert/blob/master/deployment/docker-swarm/standalone/docker-compose.yml#L16

Reply:
Same for Black Duck: https://github.com/blackducksoftware/hub/blob/v2019.12.1/docker-swarm/docker-compose.yml#L125

:exploding_head: